### PR TITLE
Update travis with support python versions

### PR DESCRIPTION
**Pull request by insacuri**
## What? and Why?
Since upgrading everything to 3.6, we no longer need to test 3.4 and 3.5.
Also added 3.7 for when we eventually upgrade to it so we can have confidence that it works

## Checklist
  - [ ] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ### Unreleased
+  - Added SEFT tab
```
Comment (jonnyshaw89) [inline CHANGELOG.md:2]: Where is the change for this?

Comment (insacuri) [reply CHANGELOG.md:2]: It's already in.  It got forgotten about when it was originally put in.
After this PR goes in, I'm going to take a tag for this and call it 2.0.0 as we've been fairly irresponsible with the changelog for this repo

Comment (jonnyshaw89) [reply CHANGELOG.md:2]: Woah thats a lot of changes. i hadn't expanded the view

**Review by jonnyshaw89 — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 language: python
+dist: xenial
 python:
-  - "3.6"
-  - "3.5"
+    - "3.7"
```
Comment (mfcrocker) [inline .travis.yml:4]: 🎨 dunno if intended

**Review by mfcrocker — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (insacuri) [reply .travis.yml:4]: Other sdx repos seem to be the same, though I'm not sure why...  I'll leave it in for consistency

**Pull request merged**